index.js

```diff
@@ -131,7 +133,67 @@ alexa.response = function(session) {
     this.sessionObject.clear(key);
     return this;
   };
-
+  this.audioPlayerPlay = function (url, token, playBehavior, offsetInMilliseconds, expectedPreviousToken) {
```

Comment:
Which parts are required? Sounds like it's url and playBehavior. So maybe it shouldn't be method that takes 5 parameters but only those 2 and additional options that just get merged? This way you don't need the `switch(arguments.length)` at all.